tools/ci/docker/linux/Dockerfile

```diff
@@ -285,13 +268,19 @@ RUN pip3 install cvt2utf
 RUN pip3 install cxxfilt
 RUN pip3 install esptool
 RUN pip3 install imgtool
+RUN pip3 install kconfiglib
 RUN pip3 install pexpect==4.8.0
 RUN pip3 install pyelftools
 RUN pip3 install pyserial==3.5
 RUN pip3 install pytest==6.2.5
 RUN pip3 install pytest-json==0.4.0
 RUN pip3 install pytest-ordering==0.6
 RUN pip3 install pytest-repeat==0.9.1
+# We are using the minimal python installation from the system so include
```

Comment:
Why did you move this? It is intentionally installed prior to all the other python packages.

Reply:
Just want to keep the order:), let me restore it.

Reply:
> Just want to keep the order:), let me restore it.

Ah yeah we needed these first to make sure we got wheels in the actual packages we care about. We should probably at some point just move those to be in a requirements.txt to keep them in one place and speed up the build, but that will be a little tricky with relative paths for the Dockerfile.